### PR TITLE
Upgraded to React Router V6

### LINKED ISSUE #104: ✨❓ Feature Request: Upgrade React Router to v6
**Description**

React Router has now the v6, we should update !
More informations [here](https://reactrouter.com/docs/en/v6/upgrading/v5)

### PR DESCRIPTION
**Pull request by Sam-Radnus**
# Pull Request

## Description

Upgraded to React Router V6
## Related Issue

Closes #104 

## Version
Did you remember to update the version number in the package.json file?

- [ ] For a nmajor restructuring, or large changes please bump the the major version number. (ex: 1.x.x)
- [x] For a minor change, bump the minor version number. (ex: x.1.x)
- [ ] For a bug fix, bump the patch version number. (ex: x.x.1)

## Documentation

For _any_ version updates, please verify if the [documentation page](https://docs.beccalyria.com?utm_source=github&utm_medium=pr-template) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [x] My contribution DOES require a documentation update.

### CODE REVIEW

**Review by MidouWebDev — COMMENTED** [review]
Thanks a lot @Sam-Radnus for this PR, we really appreciate your contribution !
For the health page i guess, we do have a special url for it.
I guess it is : https://status.heptagrambotproject.com/
Can you confirm ? @j-dogcoder

**Comment by j-dogcoder** [comment]
> Thanks a lot @Sam-Radnus for this PR, we really appreciate your contribution !
> For the health page i guess, we do have a special url for it.
> I guess it is : https://status.heptagrambotproject.com/
> Can you confirm ? @j-dogcoder

That is correct @MidouWebDev. I will take a look at this shortly.

**Review by j-dogcoder — APPROVED** [review]
LGTM!

File: src/Components/Footer.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
 import React from 'react';
-
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```
Comment (j-dogcoder) [inline src/Components/Footer.js:2]: Hey, please remove `Navigate & useNavigate` here, as they are unused variables.
```suggestion
import {Link} from 'react-router-dom';
```

File: src/Components/Header-Cmd.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```
Comment (j-dogcoder) [inline src/Components/Header-Cmd.js:2]: Hey, please remove Navigate & useNavigate here, as they are unused variables.

```suggestion
import {Link} from 'react-router-dom';
```

File: src/Components/Header.js [hunk 3]
```diff
@@ -2,29 +2,29 @@ import React from 'react';
 
 import HeptagramLogo from '../Assets/full-heptagram-logo.png';
 import Hamburger from './Hamburger';
-
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```
Comment (j-dogcoder) [inline src/Components/Header.js:5]: Hey, please remove Navigate & useNavigate here, as they are unused variables.

```suggestion
import {Link} from 'react-router-dom';
```

**Review by j-dogcoder — CHANGES_REQUESTED** [review]
Actually, have some small issues that need to be fixed.

File: src/Components/Header.js (hunk 3 above)
Comment (Sam-Radnus) [reply src/Components/Header.js:5]: Ok , @j-dogcoder

**Review by j-dogcoder — APPROVED** [review]

**Comment by j-dogcoder** [comment]
### 🎉 Congratulations @Sam-Radnus! 🎉 

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labeled `help wanted` for additional opportunities to contribute.

**Discord Server:**
_We would love it if you joined our community discord server! Your contribution to the project make you eligible for special roles & perks._

**Comment by Sam-Radnus** [comment]
Ok , @j-dogcoder ,I Loved working on the Website and I will make sure to join your discord server

**Comment by Sam-Radnus** [comment]
Thank You Very Much

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request merged**